src/components/CourseForm/sections/CourseFormInfo.js

```diff
@@ -55,7 +55,7 @@ export default class CourseFormInfo extends React.Component {
               name="courseNumber"
             />
           </Col>
-          <Col xs={6}>
+          <Col xs={3}>
```

Comment:
Are you confident that xs{3} is enough for the name? Splitting horizontal width equally between a string and a (smallish) integer may not be the way to go, and splitting 4:2 might work better?